### ISSUE TITLE
Game Dialogue Integration

### ISSUE THREAD

**Issue by E-Roderick** [tj-heat/signed-explorations#46]
## Description

Work on creating dialogue options for various scenarios and interactions in the game. The core of this task relates to the ideation of dialogue itself, but there is some merit to working on the inclusion of these lines within the game.

- [x] Initial dialogue creation
- [x] Game integration

## Members

- @tomwebber4398 - Dialogue creation
- @E-Roderick - Game integration

## Dependencies

- Loosely dependent on #39 
- #52

**Comment by E-Roderick** [comment]
Current dialogue (from #39):

**Dialogue 1 (Before Puzzle interface is opened)**

He doesn't seem to understand what you are saying.

**Dialogue 2 (Once puzzle menu has been opened once)**
He is pointing his nose.
I think its toward the key.

**Dialogue 3 (Upon Puzzle Complete)**
He wags his tail in excitement!

**Dialogue 4 (After Puzzle complete)**
The dog patiently waits for you.

**Comment by tomwebber4398** [comment]
++

Game control explanation
Dog Starts on the bridge now

**Comment by E-Roderick** [comment]
Removed design features and added that to new ticket #52

**Comment by E-Roderick** [comment]
@tweeeb is also a contributor

**Comment by E-Roderick** [comment]
Completed but pending additional dialogue
